src/components/structures/MatrixChat.js

```diff
@@ -578,7 +578,7 @@ module.exports = React.createClass({
                 this.setState({loggingIn: true});
                 break;
             case 'on_logged_in':
-                this._onLoggedIn(payload.teamToken, payload.isPasswordStored);
+                this._onLoggedIn(payload.teamToken);
```

Comment:
Intentional, yes. Related not really - it'll make a bit more sense when #881 lands (because that will get rid of `isPasswordStored`